src/frontend/github-oauth.ts

```diff
@@ -99,9 +124,12 @@ export default (opts: OAuthOptions): GithubOAuth => {
 			req.session.githubToken = response.data.access_token;
 
 			if (!req.session.githubToken) {
+				tracer.trace(`didn't get access token from GitHub`);
 				return next(new Error("Missing Access Token from Github OAuth Flow."));
 			}
 
+			tracer.trace(`got access token from GitHub, redirecting to ${redirectUrl}`);
+
 			req.log.info({ redirectUrl }, "Github OAuth code valid, redirecting to internal URL");
 			return res.redirect(redirectUrl);
 		} catch (e) {
```

Comment:
maybe tracer call in catch?

Reply:
added in some more tracer calls as per your comments